app/lib/error-messages.js

```diff
@@ -5,15 +5,13 @@ module.exports = {
   },
   name: {
     enterName: 'Enter the vet\'s name',
-    nameLength: 'Name must be 100 characters or fewer'
-  },
-  practice: {
-    enterName: 'Enter the vet practice name',
-    nameLength: 'Practice name must be 100 characters or fewer'
+    nameLength: 'Name must be 50 characters or fewer',
+    namePattern: "Special characters allowed include an ampersand(&), comma(,), hyphen(-), apostrophe('), space, brackets() or forward slash(/)"
   },
   urn: {
     enterUrn: 'Enter the URN',
-    urnLength: 'URN must be 100 characters or fewer'
+    urnLength: 'URN must be 50 characters or fewer',
+    urnPattern: 'Enter a valid URN'
```

Comment:
Only question I have is: I looked at the ticket to check the content is correct and can't see the content on the ticket. Has this content been signed off?

Reply:
Ah no - I knew there was something I needed to double check this morning!
Most just follow existing patterns, only potentially controversial one is the vet's name so will check that now.

Reply:
Done.